TouchSky/TouchSky/TouchSky/Views/ListTab.swift

```diff
@@ -11,23 +11,26 @@ import SlidingTabView
 struct ListTab: View {
     @State private var tabIndex = 0
     var body: some View {
-        VStack{
-            SlidingTabView(selection: $tabIndex, tabs: ["All", "Completed", "To Do"], animation: .easeInOut)
-            Spacer()
-            if tabIndex == 0 {
-                    PeakList()
-            } else if tabIndex == 1 {
-                Text("Completed")
-            } else if tabIndex == 2 {
-                Text("To Do")
+        NavigationStack {
+            VStack{
+                SlidingTabView(selection: $tabIndex, tabs: ["All", "Completed", "To Do"], animation: .easeInOut)
```

Comment:
I think I said this before, but the Tabs could be nicely represented with an enum. That enum could also be potentially responsible for filtering out your peaks, though maybe not.